Autopilot/Inc/SensorFusion.hpp

```diff
@@ -42,12 +65,22 @@ SFError_t SF_GenerateNewResult();
 
 /**
  * Get latest fused sensor data. Can be called any time data is needed after SF_init has been called once. Waits until the output struct is not being accessed by another task.
- * @return Output struct for fused data.
+ * @param [out] output Output struct for fused data.
+ * @return Error struct.
  */ 
-SFOutput_t SF_GetResult();
-
+SFError_t SF_GetResult(SFOutput_t *SFoutput);
```

Comment:
Why is there an error code here? Since this method just returns the most up to data struct, you can declare it as:

`SFOutput_t SF_GetResult();`

Reply:
On request from me. see the entry in https://app.asana.com/0/187002107997407/1200187390496034 (it's in low priority fixes)